src/hedera/hederaTypes.ts

```diff
@@ -84,3 +88,34 @@ export const asCheckAccountCreationStatus = asObject({
   status: asString,
   account_id: asOptional(asString)
 })
+
+export type SafeHbarWalletInfo = ReturnType<typeof asSafeHbarWalletInfo>
+export const asSafeHbarWalletInfo = asSafeCommonWalletInfo
+
+export interface HbarPrivateKeys {
+  mnemonic?: string
+  privateKey: string
+}
```

Comment:
You call these types "Hbar", but we use "Hedera" almost everywhere else.

While we haven't been 100% consistent on this, I think we need to name things after the network, not the currency. This is because people can split and clone networks, creating lots of currencies (ETH / ETC), but the engine stays the same. So naming types after the coin is short-sighted if we ever need to implement another hash-graph fork with a competing coin.

Reply:
But, that competing coin would not be called Hedera either, so then why would this be any better?

Reply:
For instance, if we call the Ethereum stuff EthEngine and so forth, then it wouldn't make sense for Ethereum Classic, since that's "ETC", not "ETH". However, it's still an Ethereum fork, and still uses things like "ERC20" for its tokens. So the network name is more sticky than the token code.

Also, all the filenames, tools, engines, and so forth already use the network name, so we should be consistent with that.